operator/src/main/java/oracle/kubernetes/operator/Main.java

```diff
@@ -840,6 +840,7 @@ public NextAction onSuccess(Packet packet, CallResponse<DomainList> callResponse
         for (Domain dom : callResponse.getResult().getItems()) {
           String domainUid = dom.getDomainUid();
           domainUids.add(domainUid);
+          boolean absent = dpis.get(domainUid) == null;
```

Comment:
Why compute absent up here, if it is used below? Unless there is a very good reason, it should be computed when you need it - you can even remove the local variable that way.

Reply:
The line after the current assignment will change the content of the map, therefore we need to catch the old value before it.